translations/en-us.yaml

```diff
@@ -100,3 +100,5 @@ reglementaireBijlageTitel:
   description: "Regulatory statement title"
   placeholder: "Untitled document"
   modify: "Modify title"
+behandelingVanAgendapunten:
+  confirmLeaveWithoutSaving: Bent u zeker dat u deze pagina wilt verlaten zonder uw wijzigingen op te slaan?
```

Comment:
change to
"Are you sure you want to leave this page without saving your changes?"

also, the key "behandelingVanAgendapuntent" (treatment of an agenda item)  doesn't make sense in this project. I'd just put it under reglementEdit (make sure to do that both in this file and in the dutch file)

Reply:
ok, sorry but didn't understand that :D

Reply:
yeah no worries you couldnt have known